reana_server/config.py

```diff
@@ -182,3 +182,6 @@ def _(x):
 # ==================
 REANA_SCHEDULER_SECONDS_TO_WAIT_FOR_REANA_READY = 1
 """How many seconds to wait between retries in case of REANA not ready to run more workflows."""
```

Comment:
One second when REANA is not ready is perhaps too soon?  Imagine getting RAM information etc... Quite a few checks to perform.  I guess we could set it to 60 seconds too, as the delaying?

Reply:
BTW we may want to expose these to REANA cluster admins so that the values would be configurable via Helm values. (This could be also done after merge via separate issue.)

Reply:
>BTW we may want to expose these to REANA cluster admins so that the values would be configurable via Helm values. (This could be also done after merge via separate issue.)

Mentioned in [this issue](https://github.com/reanahub/reana/issues/509), could be fixed in one go

Reply:
I'm probably wrong but would this be blocking the scheduler for 60 seconds?
What is the difference between `REANA_SCHEDULER_SECONDS_TO_WAIT_FOR_REANA_READY` and `REANA_SCHEDULER_RETRY_DELAY`?

Reply:
For production-like conditions we could use 60 seconds retry and 600 seconds queueing time; for developer-like conditions we could use 6 seconds retry and 60 seconds queueing time perhaps?

Reply:
Blocking the scheduler for 60 seconds if a workflow gets rescheduled in a dev environment seems quite a lot. Or perhaps I'm not understanding the working properly.